docs/README.md

```diff
@@ -1,16 +1,22 @@
 ---
 home: true
-heroImage: http://logo.factorial.io/color.svg
+heroImage: https://logo.factorial.io/color.svg
 heroText: "Phabalicious"
-tagline: A deployment system and general purpose helper.
+tagline: Supports your deployments and every-day devops-tasks
 actionText: Get Started →
 actionLink: /guide/
 features:
-  - title: Every-day tasks
-    details: Reset, backup or deploy your application, copy data from one instance to another.
-  - title: Docker integration
-    details: Administer your docker based stack, local or remote.
-  - title: Scaffolding
-    details: Scaffold new projects, create or destroy projects e.g with your CI.
+  - title: All your instances
```

Comment:
Just a comment:

Am wondering what is the difference between, 

- instances
- app
- project

Maybe it would be a nice idea to addd a glossary.